src/Package/MSBuild.Engine.Corext/MsBuild.Engine.Corext.nuspec

```diff
@@ -29,8 +29,6 @@
     <file src="$X86BinPath$/Microsoft.Build.Tasks.Core.dll" target="v15.0/bin" />
     <file src="$X86BinPath$/Microsoft.Build.Utilities.Core.dll" target="v15.0/bin" />
     <file src="$X86BinPath$/System.Collections.Immutable.dll" target="v15.0/bin" />
-    <file src="$X86BinPath$/System.IO.Compression.dll" target="v15.0/bin" />
-    <file src="$X86BinPath$/System.Runtime.InteropServices.RuntimeInformation.dll" target="v15.0/bin" />
```

Comment:
This has implications for internal consumers, right? Has the internal cloud builder image moved to 4.7.2?

Reply:
Last time we talked they seemed OK with it.